frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableContent.component.tsx

```diff
@@ -73,7 +76,16 @@ export const TicketsTableContent = ({ setSidePanelData }: TicketsTableContentPro
 		);
 	}
 
-	if (groupBy === NONE_OPTION) return (<TicketsTableGroup ticketsWithModelId={filteredItems} onNewTicket={onGroupNewTicket('')} onEditTicket={setSidePanelData} />);
+	if (groupBy === NONE_OPTION) {
+		return (
+			<TicketsTableGroup
+				ticketsWithModelId={filteredItems}
+				onNewTicket={onGroupNewTicket('')}
+				onEditTicket={setSidePanelData}
+				selectedTicketId={selectedTicketId}
```

Comment:
Would it be best to use a context instead of passing selectedTicketId down through components?

Reply:
It's really just one component that does not need it (`TicketsTable`), yet receives it and passes it down to the children
Creating a context just for that one property seems an overkill tbh